c_src/sqlite3_nif.c

```diff
@@ -691,6 +691,112 @@ exqlite_step(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
     }
 }
 
+static ERL_NIF_TERM
+exqlite_insert_all(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{
+    statement_t* statement;
+    if (!enif_get_resource(env, argv[0], statement_type, (void**)&statement)) {
+        return raise_badarg(env, argv[0]);
+    }
+
+    int stmt_param_count = sqlite3_bind_parameter_count(statement->statement);
+    int types_array[stmt_param_count];
```

Comment:
This needs to be dynamically allocated. It won't work with the windows compiler.

Reply:
Ah, I see. I'll fix it properly tomorrow!

Reply:
I wonder if the Zig compiler would be a better long term option for Windows. 🤔 I'll have to play with that idea later.

Reply:
@ruslandoga this is the last bit that we need to update. Otherwise this looks good to me.

Reply:
👋 @warmwaffles 

Right, sorry. Got a bit distracted again. I'll finish it up this weekend!

Reply:
I pushed a commit but I don't like it ...

> I wonder if the Zig compiler would be a better long term option for Windows.

Maybe just Clang? It seems to be supported on Windows, and can be use for pre-compilation. We can also document that Windows users, if they want to compile the library from scratch, should use Clang.

It seems like too much effort trying to adapt to different compilers :)

Reply:
Side issue: there seems to be something wrong with Makefile, it doesn't recompile on sqlite_nif.c changes.

Reply:
Seems to be `cc_precompiler` issue, disabling it makes `mix compile` recompile touched files.